document/core/syntax/instructions.rst

```diff
@@ -467,6 +467,14 @@ Instructions in this group are concerned with linear :ref:`memory <syntax-mem>`.
      \K{v128.}\LOAD\K{64\_splat}~\memarg \\&&|&
      \K{v128.}\LOAD\K{32\_zero}~\memarg ~|~
      \K{v128.}\LOAD\K{64\_zero}~\memarg \\&&|&
+     \K{v128.}\LOAD\K{8\_lane}~\memarg ~|~
+     \K{v128.}\LOAD\K{16\_lane}~\memarg \\&&|&
+     \K{v128.}\LOAD\K{32\_lane}~\memarg ~|~
+     \K{v128.}\LOAD\K{64\_lane}~\memarg \\&&|&
+     \K{v128.}\STORE\K{8\_lane}~\memarg ~|~
+     \K{v128.}\STORE\K{16\_lane}~\memarg \\&&|&
+     \K{v128.}\STORE\K{32\_lane}~\memarg ~|~
+     \K{v128.}\STORE\K{64\_lane}~\memarg \\&&|&
```

Comment:
Can we avoid all the repetition (including splat) by defining an auxiliary lanewidth non-terminal?

Reply:
Good idea, it does look a tiny bit squished up: 
<img width="736" alt="Screen Shot 2021-02-04 at 9 28 43 AM" src="https://user-images.githubusercontent.com/1749303/106931418-6ee7a080-66cb-11eb-9e67-5b89362fb54d.png">

(Also made a fix to the load/store lane instructions, they take a `\laneidx` immediate after the `\memarg`.